spec/paths/v1@token_info@.yaml

```diff
@@ -0,0 +1,16 @@
+get:
+  operationId: getTokenInfo
+  summary: Get your token information
+  description: Gives token information
```

Comment:
We can be more specific in `summary` and `description` of APIs. Like `API to get resource information associated with you token like your enterprise, user etc`

Reply:
done